Makefile

```diff
@@ -18,6 +18,9 @@ SOURCE := $(wildcard src/*.c)
 
 EXECUTABLE := uvb-server
 
+DB_PREFIX := /var/db
```

Comment:
I considered allowing it to be set from an environment variable-- thoughts on that?